assets/js/multiplayer.js

```diff
@@ -113,11 +113,13 @@ socket.on("player1", function(data) {
 //Player2 Joined Game Listener
 socket.on("player2", function(data) {
     var message = "Hello , " + playerName;
+    $("#opposite-player-name").html(playerName);
     $("#msg").html(message);
 });
 //Update the Creater's Name Listener
 socket.on("welcomeGame", function(data) {
     $("#player1Name").html(data);
+    $("#opposite-player-name").html(data);
```

Comment:
@ChintanPalan "welcomeGame" is a broadcast (from "joinedGame"), so all listeners will set the same #opposite-player-name.

Reply:
Ohh, got that wrong! Thanks for the clarification.